runtime-modules/working-group/src/errors.rs

```diff
@@ -84,6 +84,12 @@ decl_error! {
         /// Worker exit rationale text is too short.
         WorkerExitRationaleTextTooShort,
 
+        /// Worker storage text is too long.
+        WorkerStorageTextTooLong,
```

Comment:
Maybe change these two error names to:
```
WorkerStorageValueTooLong,
WorkerStorageValueTooShort,
```

Reply:
Good, point i also removed an error for lower bound limit https://github.com/Joystream/joystream/pull/2180/commits/33f53545ed1b9cff1ceb7adc3a0b4a705766ce68